### DISCUSSION TITLE
[Accelerate] MongoDB with multiple databases

### DISCUSSION THREAD

**Discussion by bnsouza** [prisma/prisma#19055] [Q&A] [answered]
Hi everyone! How are you doing?

I was creating my first project on Accelerate and I didn't saw that coming... I am using MongoDB to this project and I have multiple databases... I learned I can use the `datasources: {db: {url}}` to dynamically set my database in Prisma, but in Accelerate and I don't know if I can use it right now or if there is a way to solve this (since the URL created by Accelerate use the database and there isn't ways to change it after).

Can I create a project using a default database and after that change to database I really need to work? Do I have to create a link to each database? There is another solution to this? Thank you very much!!!

**Comment by nurul3101** [comment, marked as answer]
Hey @bnsouza 👋 

At the moment, it's not possible to update the database connection string for an Accelerate project (Early Access), so you would need to create different projects that connect to different databases.
